components/tools/OmeroWeb/omeroweb/decorators.py

```diff
@@ -457,6 +457,9 @@ def wrapped(request, *args, **kwargs):
                     ctx.load_server_settings(conn, request)
 
                     share_id = kwargs.get('share_id')
+                    if share_id is None:
```

Comment:
With the 'images' api, all IDs are passed in the query string. We just need to use the share_id in the decorator to get a share connection so that the share member can get a share connection to list the images in a share in the tree. 
https://github.com/will-moore/openmicroscopy/commit/dc06b68ca571c5142340a72ee0c94704a80b8f47